apps/spruce/src/pages/version/taskDuration/TaskDurationTable.tsx

```diff
@@ -24,6 +26,8 @@ import { TaskDurationCell } from "./TaskDurationCell";
 const { getDefaultOptions: getDefaultFiltering } = ColumnFiltering;
 const { getDefaultOptions: getDefaultSorting } = RowSorting;
 
+type TaskDurationColumnData =
+  VersionTaskDurationsQuery["version"]["tasks"]["data"][0];
 interface Props {
   tasks: VersionTaskDurationsQuery["version"]["tasks"]["data"];
```

Comment:
This can be typed as 
```suggestion
  tasks: TaskDurationColumnData[];
```